autoload/codefmt/xmllint.vim

```diff
@@ -0,0 +1,72 @@
+" Copyright 2017 Google Inc. All rights reserved.
+"
+" Licensed under the Apache License, Version 2.0 (the "License");
+" you may not use this file except in compliance with the License.
+" You may obtain a copy of the License at
+"
+"     http://www.apache.org/licenses/LICENSE-2.0
+"
+" Unless required by applicable law or agreed to in writing, software
+" distributed under the License is distributed on an "AS IS" BASIS,
+" WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+" See the License for the specific language governing permissions and
+" limitations under the License.
+
+
+let s:plugin = maktaba#plugin#Get('codefmt')
+
+""
+" @private
+"
+" Formatter provider for Bazel BUILD files using xmllint.
+function! codefmt#xmllint#GetFormatter() abort
+  let l:formatter = {
+      \ 'name': 'xmllint',
+      \ 'setup_instructions': 'Install xmllint. ' .
+          \ '(man xmllint).'}
```

Comment:
"man xmllint" isn't too helpful as installation instructions. It'd only do anything if xmllint were _already_ installed. Could you point to a website with info about how to install on various platforms?